tools/generate_datafile/tf_dataset_converter/datasets.py

```diff
@@ -9,7 +9,7 @@
 
 class Mnist():
     def check(dataset_name):
-        supported_dataset_list = ['fashion_mnist']
+        supported_dataset_list = ['fashion_mnist', 'mnist']
```

Comment:
* about TensorFlow datasets mnist dataset : https://www.tensorflow.org/datasets/catalog/mnist 

AFAIU, this tool helps preprocess(e.g normalize, one-hot encode) data and save it to the binary files. 
Like `fashion-mnist` dataset, the newly added `mnist` dataset needed the same preprocess.

So, Let's add `mnist` here to use mnist dataset on onert_train.